backend/kernelCI_app/constants/ingester.py

```diff
@@ -48,3 +48,9 @@
 except (ValueError, TypeError):
     logger.warning("Invalid INGEST_QUEUE_MAXSIZE, using default 5000")
     INGEST_QUEUE_MAXSIZE = 5000
+
+
+PRIO_DB = is_boolean_or_string_true(os.environ.get("PRIO_DB", "True"))
```

Comment:
I think adding PRIO_DB is bringing a lot of complexity to this pull request. If we decide to change the policy again in the future, we can send another PR.
I believe we can assume you aren't going to need it, though.